packages/cli/package.json

```diff
@@ -108,6 +110,9 @@
     "stack-trace": "0.0.10",
     "style-loader": "^0.23.1",
     "sw-precache-webpack-plugin": "^0.11.2",
+    "terser-webpack-plugin": "^1.1.0",
+    "typescript": "^3.3.3333",
+    "unfetch": "^4.0.1",
```

Comment:
Just curious, what are we doing with unfetch ?

Reply:
good question, no idea where that came from, probably something old that slipped in